tools/cdms/README.md

```diff
@@ -14,7 +14,7 @@ Imported packages:
 * csv
 * collections
 * logging
-    
+* xarray   

```

Comment:
@alovett-COAPS It appears xarray isn't used at all in cdms_reader.py so I think we should remove it as a dependency.